CONTRIBUTING.md

```diff
@@ -102,32 +104,24 @@ Here are some things to keep in mind as you file pull requests to fix bugs, add
 
 ### Release process
 
-When updating a dependency, we also need to make sure that the package is updated in all of the modules where
-the packages is needed. To do this, run:
-
-```
-bolt upgrade {package-name}@latest
-
-// i.e.
-bolt upgrade electron-wix-msi@latest
-```
-
-### Release process
-
-- Make sure the tests pass
-- `$ ./tools/bump.ts $NEW_VERSION`
-  - This will commit the changes automatically. Run `git log` to confirm that the changes have been
-    committed.
-  - `$NEW_VERSION` should be an un-prefixed [semantic version](https://semver.org/) number (e.g. `6.0.0-beta.67)
+- Make sure the tests pass with `lerna run test`
 - Run `git clean -fdx` - this will ensure unneeded build files (and potentially sensitive files) are not included in the npm package.
-- `$ node tools/publish.js`
-  - Ensure that you are logged into npm via command line (`npm login`)
+- Ensure that you are logged into npm via command line (`npm login`)
+- `lerna publish --force-publish`
+
+  - running this command without the `--force-publish` flag will only publish packages that have changes since
```

Comment:
Can we remove the whitespace here, and add an alert sign/bold to this, just so people don't miss this?